### PR TITLE
Fix J110

### PR DESCRIPTION
**Pull request by nabijaczleweli**

### CODE REVIEW

**Comment by thecoshman** [comment]
I'm going to rethink these JUMP options a bit more first...

File: ISA.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The 'name' is for either a group or single instruction.
 Bit Mask  | Name | Count | Description
 ----------|------|-------|------------
 0000 XXXX |      |    16 | Reserved
-0001 0XXX | JUMP |     8 | Jump, see section below
+0001 0FFF | JUMP |     8 | Jump, see section below
```
Comment (thecoshman) [inline ISA.md]: Don't like this change. `X` is standard way of saying you don't really care about the state of this bit. The other instructions follow this, except where they are immediately describing what those bits signify.

File: ISA.md [hunk 2]
```diff
@@ -135,7 +135,5 @@ FFF | Name | Description
 011 | JMPC | Carry flag
 100 | JMZG | Zero OR Greater than flags
 101 | JMZL | Zero OR NOT Greater than flag
-110 |      | Unconditional Jump PUSH return address (always jumps, will use the S & F registers to hold the target address - this will need be clear manually)
+110 | JMLT | NOT Zero AND NOT Greater than flag (i.e. less than)
```
Comment (thecoshman) [inline ISA.md]: This opcode should named `JMPL` to match `JMPG`

File: ISA.md (hunk 1 above)
Comment (nabijaczleweli) [reply ISA.md]: Fixed

File: ISA.md (hunk 2 above)
Comment (nabijaczleweli) [reply ISA.md]: Fixed

**Comment by thecoshman** [comment]
Maybe `JMPG` should also clearly state that the `Zero` flag should be unset as well. Though I think the compare instructions does explain that you can't be 'zero and greater', it's obviously not a logical state. I don't know if I make that quite clear enough in the compare instructions description, I think I do.

**Comment by nabijaczleweli** [comment]
There's nothing stopping you from setting both Z and G, so that needed be specified. Fixed.

**Pull request merged**